toolkit/scripts/containerized-build.mk

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Contains:
+#	- Container env for building RPMs
+
+######## CONTAINER ENV ########
+
+# General targets
+.PHONY: containerized-rpmbuild containerized-rpmbuild-help
+
+containerized_build_args :=
+ifneq ($(MODE),)
+containerized_build_args += -m ${MODE}
+endif
+
+ifneq ($(REPO_PATH),)
+containerized_build_args += -p ${REPO_PATH}
+endif
+
+ifneq ($(VERSION),)
+containerized_build_args += -v ${VERSION}
+endif
+
+ifneq ($(MOUNTS),)
+containerized_build_args += -mo ${MOUNTS}
```

Comment:
You seem to be mixing parameter name styles. `--enable-local-repo` is a multi-character param that uses two dashes. But `-mo` is a multi-character param that uses one dash.